bears/general/SpaceConsistencyBear.py

```diff
@@ -3,20 +3,40 @@
 from coalib.bears.LocalBear import LocalBear
 from coalib.results.Diff import Diff
 from coalib.results.Result import Result
+from coalib.bearlib.aspects import map_setting_to_aspect
+from coalib.bearlib.aspects.Formatting import (
+    Formatting,
+    TrailingSpace,
+    Indentation,
+    NewlineAtEOF,
+)
 
 
-class SpaceConsistencyBear(LocalBear):
+class SpaceConsistencyBear(
+        LocalBear,
+        aspects={
+            'detect': [
+                Indentation,
+                NewlineAtEOF,
+                TrailingSpace,
+            ]},
+        languages=['Python']):
     LANGUAGES = {'All'}
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
     CAN_FIX = {'Formatting'}
 
     @deprecate_settings(indent_size='tab_width')
+    @map_setting_to_aspect(
+        enforce_newline_at_EOF=NewlineAtEOF.newline_at_EOF,
+        allow_trailing_whitespace=TrailingSpace.allow_trailing_spaces,
+        indent_size=Indentation.indent_size,
+        use_spaces=Indentation.indent_type == 'space',)
```

Comment:
Try to write tests for the above mappings, running some `SpaceConsistencyBear` analysis on test data based on those aspects and tastes. Of course, the `use_spaces` setting won't work yet due to missing support for that mapping expression in `@map_setting_to_aspect`. But better a temporarily failing test due to missing preconditions than no verification at all that the above mapping stuff is actually working. It doesn't make sense to merge this before

cc @coala/aspects-developers

Reply:
@userzimmermann Why can't we have `use_spaces` as a taste of the Indentation aspect rather than `indent_type`?  (https://github.com/coala/coala/blob/master/coalib/bearlib/aspects/Formatting.py#L162)
Obviously this wouldn't mean dodging the improvements to the mapping decorator, it just seems like a more logical approach that would save some unnecessary trouble